### PR TITLE
Embed MapFilter component

### PR DESCRIPTION
**Pull request by sethfitz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "main": "app.js",
   "scripts": {
     "lint": "standard",
-    "start": "electron app.js --disable-http-cache",
+    "start": "browserify src/index.js > bundle.js && electron app.js --disable-http-cache",
```
Comment (sethfitz) [inline package.json]: @noffle what's a more appropriate target location for `bundle.js`?

Comment (gmaclennan) [reply package.json]: Actually, electron exposes require, so we don't need to bundle here.

Comment (sethfitz) [reply package.json]: We do, partially for Mapbox GL (to remove the flow type annotations). I tried pointing at the `dist` build (https://github.com/mapbox/mapbox-gl-js#using-mapbox-gl-js-with-other-module-systems), but it still hung up in the same spot.

Bundling also addressed a problem with my use of `Object.entries` and removed a few extraneous deps that were probably coming from Mapbox GL (`jsdom`, `gl`, and `sinon`).

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Mapfilter Desktop
+# MapFilter Desktop
```
Comment (hackergrrl) [inline README.md:1]: :+1:

File: app.js [hunk 3]
```diff
@@ -22,6 +22,7 @@ var userDataPath = app.getPath('userData')
 function parseArguments (args) {
   return minimist(args, {
     default: {
+      // TODO unused
```
Comment (hackergrrl) [inline app.js]: Please feel free to strip this out entirely.

File: package.json (hunk 1 above)
Comment (gmaclennan) [reply package.json]: ok good to see we're not the only ones struggling with how to publish modules that work with different bundlers...

Comment (hackergrrl) [reply package.json]: @mojodna You could put it in `build/`.

File: package.json [hunk 4]
```diff
@@ -43,6 +45,13 @@
     "appdmg": "0.4.5"
   },
   "devDependencies": {
+    "babel-plugin-transform-class-properties": "^6.19.0",
```
Comment (hackergrrl) [inline package.json]: Is Babel a hard dep of React? Is there any way we can avoid pulling babel in?

**Review by hackergrrl — CHANGES_REQUESTED** [review]

File: app.js (hunk 3 above)
Comment (sethfitz) [reply app.js]: All of the args, or just the port part?

File: package.json (hunk 4 above)
Comment (sethfitz) [reply package.json]: No, definitely not. I'll remove as much as I'm able; I may need to tweak `react-mapfilter`'s Babel config to correctly transpile `Object.entries`.

File: app.js (hunk 3 above)
Comment (hackergrrl) [reply app.js]: Just the ports.

File: package.json (hunk 4 above)
Comment (gmaclennan) [reply package.json]: I think I pushed a change last night which should have included the polyfill

**Review by hackergrrl — APPROVED** [review]

File: package.json (hunk 4 above)
Comment (sethfitz) [reply package.json]: I saw that, but it's not quite working properly yet. I'll leave this in for now and revisit later.

**Pull request merged**